spec/system/v2_lesson_project_submissions/add_submission_spec.rb

```diff
@@ -31,26 +31,29 @@
     end
 
     context 'when setting a submission as private' do
-      it 'will display the submission for the submission owner but not for other users' do
-        wait_for_turbo_frame("project-submissions_lesson_#{lesson.id}") do
-          form = Pages::ProjectSubmissions::Form.new.open.fill_in
-          form.v2_make_private
-          form.submit
-        end
+      30.times do
```

Comment:
This is just temporary and will be removed before merging. I want to  🔨  this test in CI to make sure this is fixed.

Reply:
Looks like it's still failing 😔